Lesson26/Kubra/script.js

```diff
@@ -28,17 +21,38 @@ function fetchPosts() {
 				paragraph.innerText = post.body;
 				listItem.appendChild(paragraph);
 
+				const editButton = document.createElement('a');
+				editButton.classList.add('button button--danger button--edit');
+				editButton.textContent = "Edit";
+				editButton.href = `./edit-post/edit-post.html?postId=${post.id}`;
+				listItem.appendChild(editButton);
+
+				const deleteButton = document.createElement('a');
+				deleteButton.classList.add('button button--danger button--delete');
```

Comment:
Minor issue here, can be fixed like this:
```suggestion
				deleteButton.classList.add('button', 'button--danger','button--delete');
```
Different class names have to be passed as array of strings to be processed correctly.